packages/common-scripts/tests/p2pkh.test.ts

```diff
@@ -0,0 +1,95 @@
+import test from "ava";
+import { Script, utils } from "@ckb-lumos/base";
+import { Reader } from "@ckb-lumos/toolkit";
+import { default as createKeccak } from "keccak";
+import { createP2PKHMessageGroup } from "../src/p2pkh";
+import { txObject, txSkeletonFromJson } from "./helper";
+import p2pkhJson from "./p2pkh.json";
+
+test("omni lock [g1]", (t) => {
+  const SIGNATURE_PLACEHOLDER = new Reader("0x" + "00".repeat(85));
+  let tx = txSkeletonFromJson(
+    p2pkhJson["OMNI_LOCK_[G1]"].tx as txObject,
+    SIGNATURE_PLACEHOLDER
+  );
+
+  const hasher = new utils.CKBHasher();
+  const signLock = p2pkhJson["OMNI_LOCK_[G1]"].SIGN_LOCK as Script;
+
+  const messageGroup = createP2PKHMessageGroup(tx, [signLock], {
+    hasher: {
```

Comment:
why not use default hasher here?

Reply:
No reason. I think we are still allowed to pass in default value and it won't affect performance.